intl/en/messages.json

```diff
@@ -0,0 +1,3 @@
+{
+  "22b8f1a3c27ad6df47f37a733dc8bd7f": "method is required when calling invoke()"
+}
```

Comment:
please re-run `slt-globalize -e` to get the latest strings every time you make change to the translatable strings